certificate/certificate.go

```diff
@@ -223,6 +236,27 @@ func (c Certificate) GetBooleanValidity() (trusted_ubuntu, trusted_mozilla, trus
 	return
 }
 
+// GetValidityMap converts boolean validity variables to a validity map.
+func GetValidityMap(trusted_ubuntu, trusted_mozilla, trusted_microsoft, trusted_apple, trusted_android bool) map[string]ValidationInfo {
+
+	vUbuntu := ValidationInfo{IsValid: trusted_ubuntu}
+	vMozilla := ValidationInfo{IsValid: trusted_mozilla}
+	vMicrosoft := ValidationInfo{IsValid: trusted_microsoft}
+	vApple := ValidationInfo{IsValid: trusted_apple}
+	vAndroid := ValidationInfo{IsValid: trusted_android}
+
+	m := make(map[string]ValidationInfo)
+
+	m[ubuntu_TS_name] = vUbuntu
+	m[mozilla_TS_name] = vMozilla
+	m[microsoft_TS_name] = vMicrosoft
+	m[apple_TS_name] = vApple
+	m[android_TS_name] = vAndroid
+
```

Comment:
I'm a bit confused... is this just converting the input trusted booleans into a map without doing any other computation? If so, that looks like something that should be handled with structs instead.